android/build.gradle

```diff
@@ -12,13 +12,15 @@ buildscript {
 
     dependencies {
       classpath("com.android.tools.build:gradle:7.0.4")
+      classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${safeExtGet('kotlinVersion', '1.6.10')}"
```

Comment:
Not the most important thing, but kotlin is at 1.7.0 now, any reason to pick 1.6.10 as the fallback specifically, or could we use 1.7.0 so we're at least current to start?

Reply:
will stick with this for now - I want to get this in prior to any work updating to new SDK #175